.github/workflows/integrations-tests.yml

```diff
@@ -15,8 +15,8 @@ jobs:
           run_daemon: true
       - uses: actions/setup-node@v1
         with:
-          node-version: '16.x'
-      - run: yarn install --frozen-lockfile --ignore-scripts
+          node-version: '20.x'
```

Comment:
For other scripts in this directory, it uses node v18 but this files used v20, is there a reason?

Reply:
We have packages used for tests that fail on the `18.x` version.

Reply:
Then lets upgrade everything to v20 else we won't be sure if they really work for v18 ?

Reply:
SDK works fine with `18.x`, as far as I remember, the only module that doesn't is IPFS client which isn't part of the SDK (the issue might occur due to the dynamic import).
On node `18.x`, tests involving IPFS client, fail with:
```
A jest worker process (pid=3634) was terminated by another process: signal=SIGSEGV, exitCode=null. Operating system logs may contain more information on why this occurred.
```
https://github.com/docknetwork/sdk/actions/runs/8281616475/job/22661409906#step:7:2563

Reply:
Its fine for the moment but in future we might not learn when tests don't work on v18